src/state/auth/KeycloakManager.ts

```diff
@@ -0,0 +1,17 @@
+import Keycloak from 'keycloak-js';
+
+const keycloakInstance = Keycloak({
+    url: 'http://localhost:8089/auth/',
```

Comment:
Maybe create a CoMPAS Setting from this directly?

Remark: I think you should first merge the main branch, because it looks that the last change to setting it's in this branch.